docs/models.rst

```diff
@@ -79,3 +79,19 @@ It is also possible to have two concrete Heroku Connect models to Inherit from
 each other.
 
 .. _`Multi-table inheritance`: https://docs.djangoproject.com/en/stable/topics/db/models/#multi-table-inheritance
+
+
+Signals
+-------
+
+Standard `Django model signals`_ won't be triggered in case of changes coming from Salesforce. This is happening because
+Heroku Connect operates on a database level.
+If you want to hook up to changes coming from Salesforce, there are several possible solutions:
+
+* Use `Salesforce PushTopic Events`_
+* Set up `Salesforce Outbound message`_ (triggered on object change) and a Django SOAP endpoint listening to it. From there You could make a Django signal to hook up.
+* Use database triggers and set up a a task queue listening to it.
```

Comment:
There isn't really a task queue in PostgreSQL. However, you would use an insert and update trigger and use `pg_notify` to have a consumer listen and perform actions.
```suggestion
* Use database triggers and set up a task queue listening to it.
```

Reply:
@codingjoe postgres `LISTEN` / `NOTIFY` has a queue ( see https://www.postgresql.org/docs/13/sql-notify.html ).

Reply:
@amureki I would mention the postgres techniques that would be used in that sentence (the ones in the comment before)

Reply:
@codingjoe @syphar what do you think about this phrasing? 

>* Use PostgreSQL ``LISTEN`` and ``NOTIFY`` features. See related `documentation`_ and `an example Python library`_